apps/web/src/components/Explore/index.tsx

```diff
@@ -25,6 +25,15 @@ const Explore: NextPage = () => {
   const router = useRouter();
   const currentProfile = useProfileStore((state) => state.currentProfile);
   const [focus, setFocus] = useState<PublicationMetadataMainFocusType>();
+  const searchParams = useSearchParams();
+
+  const [tabIndex, setTabIndex] = useState(
+    Number(searchParams.get('tab')) || 0
+  );
+
+  useEffect(() => {
+    router.replace(`?tab=${tabIndex}`);
+  }, [router, tabIndex]);
```

Comment:
why we need to do this? can we make it default instead, this updates the URL everytime, it makes bit ugly 🙈

Reply:
That's the default behaviour in even in pages branch and is recommended for UX.

Reply:
https://react.dev/learn/you-might-not-need-an-effect

Try avoiding effect unless its doing an external network call, router replace is an expensive operation as well 😕 

I will say nuke this!